examples/example-all-pass/expected_results.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version" : "v2",
+  "status" : "pass",
+  "message" : "3 ran, 3 passed.",
```

Comment:
This key should not be present for the `pass` state.

Reply:
@ErikSchierboom: Just to confirm: Top-level "message" key should be just for error status present?

Reply:
E.g.: If I have 3 tests: one passes, one fails, one raises error (e.g. exception). What should be top-level error message (if ever)?

Reply:
None. So the best way to think about this is as follows:

- If no tests could be executed, for example due to a syntax error or something like that, then the top-level `status` field is `error` and and the `message` should be set and describe the compiler/runtime error
- If any tests could be executed, the top level `status` field will be `pass` or `fail` and no `message` property set

Speaking of errors, if a test fails because the code threw an error/exception, that is usually still a `fail` status for the test. Applying the above logic here, the `error` status should only be used if tests could not be executed. Throwing an error is usually still reported as a failure by the test framework, so that would count as a `fail` test.

Reply:
@ErikSchierboom Well, SUnit is quite strict about this distinction, library consider failures only if assertion fails. Errors are considered all raised errors and exceptions during runtime (e.g. division by 0), so that assertion could not be successfully evaluated. I don't think I can influence this (it would mean some tailored modification of SUnit, which isn't worth of it I think).

Reply:
Anyway, I will modify top-level error logic, so that only syntax errors during code compilation would put error output.

Reply:
> I don't think I can influence this (it would mean some tailored modification of SUnit, which isn't worth of it I think).

Don't bother, it's fine